nidaqmx/tests/test_events.py

```diff
@@ -76,3 +76,66 @@ def callback(task_handle, every_n_samples_event_type,
             task.stop()
 
             assert non_local_var['samples_read'] == 0
+    
+    @pytest.mark.parametrize('seed', [generate_random_seed()])
+    def test_signal_event(self, x_series_device, seed):
```

Comment:
Since this unit test doesn't actually test the feature in question, I would revert this for now. I can merge your changes to correct the library used for the `DAQmxRegisterSignalEvent` function, though, thanks!

Reply:
thats fine for me - actually I was just too lazy to do the Unittest right.